### PR TITLE
allowed lines and markers in data explorer scatterplot

### PR DESCRIPTION
**Pull request by shivasaurus**
Connects #3774

### CODE REVIEW

File: src/app/log-tool/visualize/visualize-menu/graph-basics/graph-basics.component.ts [hunk 1]
```diff
@@ -32,6 +34,8 @@ export class GraphBasicsComponent implements OnInit {
         }
       }
     });
+    this.markerTypes = ["lines+markers", "lines", "markers"];
```
Comment (rmroot) [inline src/app/log-tool/visualize/visualize-menu/graph-basics/graph-basics.component.ts:37]: for markerTypes do a type of {label: string, value: string} and capitalize the label for display.

File: src/app/log-tool/visualize/visualize-menu/graph-basics/graph-basics.component.html [hunk 2]
```diff
@@ -11,6 +11,11 @@
             <option *ngFor="let graphType of graphTypes" [ngValue]="graphType.value">{{graphType.label}}
             </option>
         </select>
+        <select *ngIf = "selectedGraphObj.data[0].type == 'scattergl'" class="form-control w-100" name="markerType" (ngModelChange) = "markerType = $event" [ngModel]="selectedGraphObj.data[0].mode"
+            (change)="setLinesMarkers()" (focus)="focusField()" (blur)="focusOut()">
+            <option *ngFor="let markerTypeValue of markerTypes" [ngValue]="markerTypeValue">{{markerTypeValue}}
```
Comment (rmroot) [inline src/app/log-tool/visualize/visualize-menu/graph-basics/graph-basics.component.html:16]: Put this in it's own "form-group pt-2" div

**Review by rmroot — CHANGES_REQUESTED** [review]

File: src/app/log-tool/visualize/visualize-menu/graph-basics/graph-basics.component.ts (hunk 1 above)
Comment (rmroot) [reply src/app/log-tool/visualize/visualize-menu/graph-basics/graph-basics.component.ts:37]: this.markerTypes = [{label: "Lines & Markers", value: "lines+markers"}]

**Comment by shivasaurus** [comment]
Suggested fixes have been made.

**Review by rmroot — APPROVED** [review]

**Pull request merged**